about.html

```diff
@@ -0,0 +1,151 @@
+            the crisis we rose again stronger and more ready than ever We adopted some habits as single persons and as a
+            society that after a while became our NEW NORMAL that's why StartUp Land 3 is called (The New Normal) for 3
+            days of online workshops and seminars the StartUp Land team and a group of experts will highlight the New
+            Normal in all Business related fields like - E-commerce - Marketing and Sales - Digital transformation -
+            Lean management and Business development - Investment - Opportunities for entrepreneurs - Supply Chain and
+            Logistics
+        </p>
+        <p>
+            Please contcat us per Email for any further information
+            about StartUp Land Summit 2022
+
+        </p>
+        <p class="header__Email">info@startuplandegypt.com</p>
+    </header>
+
+    <!-- main section -->
+    <main class="main" id="program">
+        <div class="main__heading">
+            <h2>Creative commons </h2>
+            <div class="main__line"></div>
+            <p>The Logo of startup land summit 2022 was decieded through logo design competition from 1.November to
+                5.December</p>
+        </div>
+        <div class="main__logo">
+            <img src="./assets/startup.jpg" alt="satrtup" />
+        </div>
+
+    </main>
+
+    <!-- times section -->
+    <section class="time">
+
+        <h2> See the past startup Global Summits</h2>
+        <div class="main__heading">
+            <div class="main__line"></div>
+        </div>
+        <p class="time__p">
+            Take a look at the last two startup
+            land Summits took place in the greek campus
+            and american university
+        </p>
+        <div class="time__container">
+            <div class="time__one">
+                <div class="time__text">
+                    <h2>2020</h2>
+                    <p>Startup land summit 2020 in American University</p>
+                </div>
+            </div>
+
+            <div class="time__two">
+                <div class="time__text">
+                    <h2>2019</h2>
+                    <p>Startup land summit 2019 in Greek campus</p>
+
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- DESKTOP: partner section -->
+    <section class="partner" id="sponsor">
+        <h2>Partner</h2>
+        <div class="main__heading">
+            <div class="main__line"></div>
+        </div>
+        <div class="partner__logos">
+            <span class="partner__logos__type1">IBM</span>
+            <span class="partner__logos__type2">Microsoft</span>
+            <span class="partner__logos__type2">Amideast</span>
+            <span class="partner__logos__type1">S PLus</span>
+            <span class="partner__logos__type1">USAID</span>
+            <span class="partner__logos__type2">Nokia</span>
+        </div>
+
+    </section>
```

Comment:
- [ ] First of all, I would like to congratulate you, your project looks AMAZING 🔥  There are some small details to make it perfect though. To match the original design better, you should consider hiding the partner section on the desktop version.
- [ ] It is ok to not have "mock logos", but the design should still be as close as possible. To achieve that, you should consider adding some `padding` for your logos, I tested with `20px` for top and bottom and it looked like the original one! You could also consider changing from `justify-content: space-around` to `center` and then add some `gap` to your logos, I think it would look so much better, don't you?

Original design:
![image](https://user-images.githubusercontent.com/66967274/147091599-3fe613f7-9755-4027-8d51-9d21757987b8.png)

![image](https://user-images.githubusercontent.com/66967274/147091558-0c6ac048-5146-43bc-b23d-dc042d24f3f4.png)

Your design:

![image](https://user-images.githubusercontent.com/66967274/147091667-9bf46ec6-03ae-4301-937f-ab872979d394.png)